ci-operator/step-registry/ipi/deprovision/vsphere/dns/ipi-deprovision-vsphere-dns-commands.sh

```diff
@@ -4,7 +4,9 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred 
+export AWS_SHARED_CREDENTIALS_FILE=${CLUSTER_PROFILE_DIR}/.awscred
+export AWS_MAX_ATTEMPTS=7
+export AWS_RETRY_MODE=adaptive
```

Comment:
https://boto3.amazonaws.com/v1/documentation/api/latest/guide/retries.html

provides details on how this mode covers the rate limit exceeded errors for retrying